docker grid/docker1/MarchBatch_2021/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+services:
+  selenium-hub:
+    image: selenium/hub:3.141.59-20210607
+    container_name: selenium-hub
+    ports:
+      - "4444:4444"
+
+  chrome:
+    image: selenium/node-chrome:3.141.59-20210607
+    volumes:
+      - /dev/shm:/dev/shm
+    depends_on:
+      - selenium-hub
+    environment:
+      - HUB_HOST=selenium-hub
+      - HUB_PORT=4444
+
+  firefox:
+    image: selenium/node-firefox:3.141.59-20210607
```

Comment:
once u added opera to the yml file make sure have it in your local 
other wise it might not show u the browsers on console 
in url localhost:4444